proto/method.go

```diff
@@ -49,35 +49,20 @@ const (
 	Scan
 	// EndTransaction either commits or aborts an ongoing transaction.
 	EndTransaction
-	// ReapQueue scans and deletes messages from a recipient message
-	// queue. ReapQueueRequest invocations must be part of an extant
-	// transaction or they fail. Returns the reaped queue messsages, up to
-	// the requested maximum. If fewer than the maximum were returned,
-	// then the queue is empty.
-	ReapQueue
```

Comment:
Nice cleanup. This thing is still mentioned in the design doc; could you remove it?